pkg/collector/corechecks/containers/containerd/containerd_events.go

```diff
@@ -55,8 +53,7 @@ func (s *subscriber) CheckEvents(ctrItf ctrUtil.ContainerdItf) {
 	ctx := context.Background()
 	ev := ctrItf.GetEvents()
 	log.Info("Starting routine to collect Containerd events ...")
-	ctxNamespace := namespaces.WithNamespace(ctx, s.Namespace)
```

Comment:
For reviewers, note that this is the correct way to specify the namespaces for most of the calls, but not for events.
For events, we need to use filters.